contracts/Core/StakeManager.sol

```diff
@@ -345,6 +347,14 @@ contract StakeManager is Initializable, ACL, StakeStorage, StateManager, Pause {
         return stakers[stakerId].epochFirstStaked;
     }
 
+    /// @return isStakerActive : Activity < Grace
+    function isStakerActive(uint32 stakerId, uint32 epoch) public view returns (bool) {
```

Comment:
This is made public so web client or CLI can access anytime if staker is active.
We can achieve so via indexing in web client as well,
But in CLI via this method, it can first check if given staker is active

Let me know if you guys think its not needed, we can make it internal

Reply:
This makes sense.